blog-server-api/src/entities/create_post.rs

```diff
@@ -0,0 +1,36 @@
+use blog_server_services::traits::post_service::BasePost;
+use serde::{Deserialize, Serialize};
+
+use crate::utils::time_utils;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CreatePost {
+    pub title: String,
+    pub slug: String,
+    pub summary: String,
+    pub published: u8,
+    pub content: Option<String>,
+    pub tags: Vec<ShortTag>,
+    pub author_id: u64, //TODO replace with authorized user ID
```

Comment:
We already can do this

Reply:
Why not?)

Reply:
was facing some complications (already discussed). now it is done